create_purge_issues.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2021 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This task creates issues in repositories using Kebechet GitHub app when data purges occur."""
+
+import os
+import logging
+
+from ogr.services.github import GithubService
+
+from thoth.storages import GraphDatabase
+from thoth.workflow_helpers import __service_version__
+
```

Comment:
we miss the metrics like in https://github.com/thoth-station/workflow-helpers/blob/5e1f52897be6980b22bcb2a173a5b278a6f931e0/graph_schema_update.py#L24 and I think it is ready!! Thanks

Reply:
Does it need send_metrics if I haven't created any prometheus metrics to push to the gateway?

Reply:
Yes, those are standard metrics, for all workflow-helpers tasks using thoth-storages :)

Reply:
I wil add them in separate PR